src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <div className="App">
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )
+          }
+      <p className="Person__partner">
+        {person.isMarried ? `${person.sex === 'm' ? `${person.partnerName} is my wife` : `${person.partnerName} is my husband`}` : 'I am not married'}
+      </p>
```

Comment:
Do not use ternary nesting. Create variable with displayed result before return JSX